src/main/java/spoon/compiler/Environment.java

```diff
@@ -344,6 +345,16 @@ void report(Processor<?> processor, Level level,
 	 */
 	File getSourceOutputDirectory();
 
+	/**
+	 * Set the output destination that handles where source files are written
+	 */
+	void setOutputDesination(OutputDestination outputDestination);
```

Comment:
The name is a bit confusing with `sourceOutputDirectory`, and for me it looks like a strategy for the final location of the output. So WDYT about renaming it to something like `setOutputStrategy` or `setOuputHandler`?

Reply:
I agree